Makefile

```diff
@@ -57,6 +57,7 @@ INTERACTIVE ?= i
 COLLECTION_BENCHMARK ?= faux
 BENCHMARK_DIR ?= /home/${UNAME}/kernmlops-benchmark
 YCSB_BENCHMARK_DIR ?= ${BENCHMARK_DIR}/ycsb
+MEMCACHED_PORT=11211
```

Comment:
please make this `?=`  that makes variable overridable by env vars so it is more configurable